slip-0044.md

```diff
@@ -1124,6 +1124,7 @@ All these constants are used as hardened derivation.
 | 2086       | 0x80000826                    | KILT    | KILT Spiritnet                    |
 | 2109       | 0x8000083d                    | SAMA    | Exosama Network                   |
 | 2112       | 0x80000840                    | IoTE    | IoTE                              |
+| 2125       | 0x80000853                    | BAY     | BitBay                            |
```

Comment:
0x853 is 2131
2125 is 0x84d

so which one is it?

Reply:
sorry, it should be  `2125`, so `0x84d` is correct.